### PR TITLE
Load CSS through webpack

### PR DESCRIPTION
**Pull request by ger-benjamin**
For GSGMF-391

Should I commit somethings elsewhere (add dependencies ?, describe the "adapt 'less' things") ? See  https://jira.camptocamp.com/browse/GSEPF-47 to know what I've done directly in the final project.

### CODE REVIEW

**Comment by ger-benjamin** [comment]
@gberaudo what about the less that I've adapted ? should it be documented ?

**Comment by gberaudo** [comment]
Yes you should document that CSS must be handled by webpack.

File: docs/how_to_migrate_from_2.2_to_2.3.md [hunk 1]
```diff
@@ -292,6 +293,16 @@ these values to the module of your controller like this:
 </script>
 ```
 
+### Adapt paths for 'less'
+
+We no more load all `.less` files in each application. You must import your `<interface.less>` in each
+application controller you have by somthing like: `import '../../less/<interface>.less'`.
```
Comment (sbrunner) [inline docs/how_to_migrate_from_2.2_to_2.3.md]: `We no more load all `.less` files in each application. ` it was already the case...

Comment (fredj) [reply docs/how_to_migrate_from_2.2_to_2.3.md]: `something` instead of `somthing`

**Pull request merged**